build/optprof/UpdateRunSettings.ps1

```diff
@@ -35,5 +37,5 @@ $testStores.AppendChild($buildDropStore)
 
 $runsettingsXml.Save($runsettingsPath)
 Write-Host 'Saved Output:'
-Write-Host "profilingInputsStore: $profilingInputsStore"
-Write-Host "buildDropStore: $buildDropStore"
\ No newline at end of file
+Write-Host "profilingInputsStore: $($profilingInputsStore.Uri)"
+Write-Host "buildDropStore: $($buildDropStore.Uri)"
```

Comment:
I had flubbed this output message from my OptProf PR. Fixing it here. No impact on functionality.